### PR TITLE
Update car_parts.json added Tiger Wheel & Tyre

### PR DESCRIPTION
**Pull request by jhutton1**
added Tiger Wheel & Tyre - https://www.twt.co.za/

### CODE REVIEW

File: data/brands/shop/car_parts.json [hunk 1]
```diff
@@ -12,6 +12,16 @@
...
   },
   "items": [
+        {
+      "displayName": "Tiger Wheel & Tyre",
+      "locationSet": {"include": ["za"]},
+      "tags": {
+        "brand": "Tiger Wheel & Tyre",
+        "brand:wikidata": "Q120762656",
+        "name": "Tiger Wheel & Tyre",
+        "shop": "car_parts"
```
Comment (Cj-Malone) [inline data/brands/shop/car_parts.json:22]: So this already exists under `shop=tyres`, but I think you might be right. Can you remove it from that file please.

Also the website suggests they do the following at all of the locations:

```suggestion
        "shop": "car_parts",
        "service:vehicle:tyres": "yes",
        "service:vehicle:car_parts": "yes",
        "service:vehicle:batteries": "yes",
        "service:vehicle:wheels": "yes"
```

Comment (JaimeHutton-TomTom) [reply data/brands/shop/car_parts.json:22]: I'll remove it from tyres.  Thanks.

Comment (jhutton1) [reply data/brands/shop/car_parts.json:22]: I'll remove it from tyres.  Thanks.

**Comment by UKChris-osm** [comment]
In cases like this @jhutton1 where you plan to also remove an item from another category, you can make the change within the same PR, rather than adding a separate one.

**Comment by Cj-Malone** [comment]
The data that feeds https://github.com/alltheplaces/alltheplaces/pull/5571, calls them:
```json
"gmbPrimaryCategoryName":"Tyre Shop",
"gmbAdditionalCategoryName1":"Wheel Shop",
"gmbAdditionalCategoryName2":"Wheel alignment service",
"gmbAdditionalCategoryName3":"Repair service",
"gmbAdditionalCategoryName4":"Car Battery Shop",
"gmbAdditionalCategoryName5":"Auto parts store",
```

OSM has 28, 20 `shop=tyres`, 6 `shop=car_repair`, [overpass](https://overpass-turbo.eu/s/1A2K). It seems to be entirely uneffected by #8426 a month ago.

I don't feel strongly towards either, although I don't know if `service:vehicle:*` tags are valid on `shop=tyres`. Which would be a shame to lose the details.

(Rel: #8615)

**Comment by AntiCompositeNumber** [comment]
Perusing their [website](https://twt.co.za), it looks like they sell
* Tyres
* Wheels
* Batteries
* Wiper blades
* Installation of 4x4 suspension kits and accessories (certain locations)
* Glass repair (certain locations)

The first two obviously fit in `shop=tyres`. Batteries and wiper blades are fairly generic items, so I don't mind them being included in a POI tagged `shop=tyres`. At locations where glass repair and 4x4 kits are being offered, `shop=car_repair` might fit better. `shop=car_parts` doesn't really seem to fit. The parts being sold are quite limited, and it seems like they're targeting sales + installation rather than direct retail sales.

I wouldn't consider the `service:vehicle` tags [invalid in any case](https://wiki.openstreetmap.org/wiki/Any_tags_you_like), just perhaps unexpected.

**Pull request closed without merging**